ansible/roles/bamboo/templates/haproxy_logrotate.j2

```diff
@@ -0,0 +1,11 @@
+{{ haproxy_log_dir }}/haproxy.log {
+    daily
+    rotate 5
+    missingok
+    notifempty
+    compress
+    delaycompress
+    postrotate
+        invoke-rc.d rsyslog rotate >/dev/null 2>&1 || true
```

Comment:
This is the logrotate.d config file that comes in when haproxy pkg gets installed. I just updated the log dir and the rotate from 52 to 5.